manifests/07-downloads-deployment.yaml

```diff
@@ -113,8 +113,11 @@ spec:
                   zip.write(path, basename)
 
           # Create socket
-          addr = ('', 8080)
-          sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+          # IPv6 should handle IPv4 passively so long as it is not bound to a
```

Comment:
@wking for a glance. This is a best guess at this one, but I don't have a clear test path (and have never written python 😄 😄).

Anything beyond this and I think I'd rather see the script built into the `cli-artifacts` image (or an image based on it) so we can work with the code more easily.

Reply:
Verified in my own non-IPv6 cluster that this still runs.